integration-tests/src/test/resources/features/webhookaction.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Webhook Action tests
```

Comment:
Please rename the file to `action-webhook.feature` to keep the consistency.

Reply:
sure